docs/section5/Rule5-34.md

```diff
@@ -0,0 +1,41 @@
+
+# Envelope - Rule 5-34  
+
+**Rule ID:** 5-34  
+**Rule Description:** If skylight area in the proposed design is 3% or less of the roof surface, the skylight area in baseline shall be equal to that in the proposed design.  
+**Rule Assertion:** B-RMR total (subsurface.glazed_area+subsurface.opaque_area) = expected value  
+**Appendix G Section:** Section G3.1-5(e) Building Envelope Modeling Requirements for the Baseline building  
+**Appendix G Section Reference:** None  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  None  
+
+**Manual Check:** None  
+**Evaluation Context:** Each Data Element  
+**Data Lookup:** None  
+**Function Call:**  
+
+  1. get_building_segment_skylight_roof_areas()  
+  2. match_data_element()
+
+## Rule Logic:
+
+- Get building skylight roof areas dictionary for B_RMR: `skylight_roof_areas_dictionary_b = get_building_segment_skylight_roof_areas(B_RMR)`
+
+- Get building skylight roof areas dictionary for P_RMR: `skylight_roof_areas_dictionary_p = get_building_segment_skylight_roof_areas(P_RMR)`
+
+- For each building segment in B_RMR: `for building_segment_b in B_RMR.building.building_segments:`  
+
+  - Calculate skylight roof ratio for building segment: `skylight_roof_ratio_b = skylight_roof_areas_dictionary_b[building_segment_b.id][0] / skylight_roof_areas_dictionary_b[building_segment_b.id][1]`
+
```

Comment:
I do not understand the nomenclature of [0] and [1], but trust that you know what you are doing here :-)

Reply:
The get_skylight_roof_areas() function will return a skylight_roof_areas_dictionary in the format of: 
building_segment_id: [total skylight area, total roof area]
[0] calls for the first value, which is the total skylight area for that building segment
[1] calls for the second, which is the total roof area for that building segment

Reply:
Got it, thanks!